docs/settings/alert-action-settings.asciidoc

```diff
@@ -59,6 +59,12 @@ You can configure the following settings in the `kibana.yml` file.
 | `xpack.actions.proxyUrl` {ess-icon}
   | Specifies the proxy URL to use, if using a proxy for actions. By default, no proxy is used.
 
+| `xpack.actions.proxyBypassHosts` {ess-icon}
+  | Specifies hostnames which should not use the proxy, if using a proxy for actions. The value is an array of hostnames as strings.  By default, all hosts will use the proxy.  The settings `xpack.actions.proxyBypassHosts` and `xpack.actions.proxyOnlyHosts` cannot be used at the same time.
+
+| `xpack.actions.proxyOnlyHosts` {ess-icon}
+  | Specifies hostnames which should only use the proxy, if using a proxy for actions. The value is an array of hostnames as strings.  By default, all hosts will use the proxy.  The settings `xpack.actions.proxyBypassHosts` and `xpack.actions.proxyOnlyHosts` cannot be used at the same time.
```

Comment:
Is there a way to make clear that hostnames not included in this configuration will not use the proxy? I did not get that from first reading this.

Reply:
Good call - I've updated the text for both of these to try to make it clearer, in commit I've added tests for these cases in commit https://github.com/elastic/kibana/pull/95365/commits/7626ecef1e66fea2db637e1fa38939b328217ac